File: LeetCode/Medium/1442-count-triplets-that-can-form-two-arrays-of-equal-xor/1442-count-triplets-that-can-form-two-arrays-of-equal-xor.java
class Solution {
    public int countTriplets(int[] arr) {
        int n = arr.length;
        int ret = 0;
        
        for (int i = 0; i < n; ++i){
            int temp = arr[i];
            
            for (int j = i + 1; j < n; ++j){
                temp ^= arr[j];
                if (temp == 0) ret += j - i;
            }
        }
        
        return ret;
    }
}

File: LeetCode/Medium/1442-count-triplets-that-can-form-two-arrays-of-equal-xor/1442-count-triplets-that-can-form-two-arrays-of-equal-xor.py
class Solution:
    def countTriplets(self, arr: List[int]) -> int:
        n = len(arr)
        ret = 0

        for i in range(n):
            temp = arr[i]
            for j in range(i+1, n):
                temp ^= arr[j]
                
                if not temp:
                    ret += j - i

        return ret